src/main/java/com/mhrglobal/print/PayslipPrintService.java

```diff
@@ -1,16 +1,21 @@
 package com.mhrglobal.print;
 
+import com.mhrglobal.clients.PrinterClient;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-//Do not change
```

Comment:
I decided to go with a different implementation that I think is much cleaner.  We delegate the task to the printer client.